regression/goto-analyzer/monotonic_change_struct/main.c

```diff
@@ -0,0 +1,25 @@
+int nondet(void);
+int nondet_bool(void);
+
+struct cartesian_coorindate
+{
+  int x;
+  int y;
+};
+
+int main()
+{
+  struct cartesian_coorindate u = {nondet(), nondet()};
```

Comment:
Would it matter if we just wrote:

```suggestion
  struct cartesian_coorindate u;
```

and perhaps remove the `nondet` declaration above?

Reply:
Good point.